components/dialog/test/dialog-fullscreen.visual-diff.html

```diff
@@ -112,5 +112,103 @@
 		<div>Bottom</div>
 	</d2l-dialog-fullscreen>
 
+	<d2l-dialog-fullscreen id="dialogSetWidth" _width="1200" title-text="Dialog Title">
```

Comment:
Note: all properties that begin with `_` are private properties, and so in this case, the mixin defines `_width` for internal use. If the `d2l-dialog-fullscreen` is expected to support a `width` property, then we should add it to the `d2l-dialog-fullscreen` component, similar to how it's done for `d2l-dialog` [here](https://github.com/BrightspaceUI/core/blob/main/components/dialog/dialog.js#L39).